app/src/components/layout/Navbar/Navbar.test.tsx

```diff
@@ -11,8 +11,40 @@ jest.mock('../../../hooks', () => {
 });
 
 describe('Navbar', () => {
+  const navItems = [
```

Comment:
Why do we have multiple instances of `navItems`?

Reply:
update!